docs/deploy-development-rp.md

```diff
@@ -131,7 +131,7 @@
 * Get an admin kubeconfig:
 
   ```bash
-  hack/get-admin-kubeconfig.sh "/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER" >admin.kubeconfig
+  hack/get-admin-kubeconfig.sh "/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER" admin >admin.kubeconfig
```

Comment:
if we go with option 1 from the PR description we can remove addional param. For now I prefer to make it explicit. 
Better yet create Makefile targets (I noticed we don't have many helper targets at the moment)